pandas/core/arrays/datetimelike.py

```diff
@@ -681,6 +673,10 @@ def searchsorted(self, value, side='left', sorter=None):
         if isinstance(value, compat.string_types):
             value = self._scalar_from_string(value)
 
+        if isinstance(value, ABCIndexClass):
+            # unbox e.g. DatetimeIndex -> DatetimeArray
+            value = value._values
```

Comment:
these lines got added for an unsuccessful attempt to make DatetimeIndex.searchsorted pass through to _data.  long-term i think that is still the right thing to do, but until then i need to revert these lines